libvast/vast/detail/vector_map.hpp

```diff
@@ -207,7 +207,12 @@ class vector_map : totally_ordered<vector_map<Key, T, Allocator, Policy>> {
 
   template <class L>
   size_type count(const L& x) const {
```

Comment:
This is no discard for vector_set, but not for vector_map. Please make that change.